packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -162,7 +162,10 @@ exports.onPostBuild = (
 
   const idbKeyvalFile = `idb-keyval-iife.min.js`
   const idbKeyvalSource = require.resolve(`idb-keyval/dist/${idbKeyvalFile}`)
-  const idbKeyvalDest = `public/${idbKeyvalFile}`
+  const idbKeyValVersioned = `idb-keyval-${
+    require(`idb-keyval/package.json`).version
+  }-iife.min.js`
```

Comment:
Please put this on a separate line, above it.
```suggestion
  const idbKeyvalPackageJson = require(`idb-keyval/package.json`)
  const idbKeyValVersioned = `idb-keyval-${idbKeyvalPackageJson.version}-iife.min.js`
```

Reply:
I applied that in https://github.com/gatsbyjs/gatsby/pull/24938/commits/0cc3df6a49e3761332fa01cc85ecd501f6249c05